Schemata/sarif-schema-2.1.0.json

```diff
@@ -2717,7 +2717,7 @@
       "properties": {
 
         "guid": {
-          "description": "A stable, unique identifier for the suprression in the form of a GUID.",
+          "description": "A stable, unique identifier for the suppression in the form of a GUID.",
```

Comment:
Please create a different PR for changes on published artifacts (on docs.oasis-open.org i.e. standards work products.